docs/int/quickstart/quickstart-mainnet.md

```diff
@@ -5,21 +5,24 @@ description: Run a cluster on mainnet
 
 # Run a DV on mainnet
```

Comment:
Not sure about the purpose of this page. Is it supposed to be a full guide to a DV config on mainnet? If so, should we add the information about DKG/create cluster. Currently looks like a guide but wouldn't be practical to follow. Alternatively we can use it as a page displaying the most important steps and configs to do on mainnet vs the other guides.

Reply:
Yeah we probably should just have one set of guides tabulated for the different networks, future problem though.